lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2388,26 +2401,21 @@ int s1ap_handle_enb_initiated_reset_ack(
     s1ap_ResetAcknowledgeIEs_p->presenceMask |=
         S1AP_RESETACKNOWLEDGEIES_UE_ASSOCIATEDLOGICALS1_CONNECTIONLISTRESACK_PRESENT;
     for (uint32_t i = 0; i < enb_reset_ack_p->num_ue; i++) {
+      sig_conn_list =
```

Comment:
Where is this memory getting freed?

Reply:
sig_conn_list gets freed up as part of function, free_s1ap_resetacknowledge(). Even I verified memory leak after execution of test case, test_enb_partial_reset_with_unknown_ue_s1ap_ids.py

Reply:
sig_conn_list gets freed up as part of function, free_s1ap_resetacknowledge(). Even I verified memory leak after execution of test case, test_enb_partial_reset_with_unknown_ue_s1ap_ids.py

Reply:
sig_conn_list gets freed up as part of function, free_s1ap_resetacknowledge(). Even I verified memory leak after execution of test case, test_enb_partial_reset_with_unknown_ue_s1ap_ids.py